wordpress-mobile/WordPress-iOS
# Revert Spectrum 22 as default app icon, back to Cool Blue

This PR reverts the temporary changes made in #18638, changing the app icon back to cool blue. I've also made sure that Spectrum 22 is still selectable in the App Icon screen.

### To test

Note: occasionally you may need to restart the simulator / device to see an app icon change. This is a longstanding issue that we've seen (including with other apps), and I think is due to some kind of caching that iOS does.

**Default app icon**

* Build and release `trunk` and ensure you have the spectrum 22 icon showing up. Don't change the icon to a custom one.
* Build and run this branch over the top. Check the icon reverts to cool blue and the correct selection shows up in App Settings > App Icon
* Check you can still manually switch to the new Spectrum 22 icon in App Settings.

**Default app icon - Jetpack**

* Build and release the **Jetpack app** from `trunk` and ensure you have the spectrum 22 icon showing up.
* Build and run this branch over the top. Check the icon reverts to the standard Jetpack icon.

**Custom app icon**

* Build and run `trunk` and check you have the spectrum 22 icon showing up. Navigate to Me > App Settings > App Icon and choose a different custom icon.
* Build and run this branch over the top. Check the icon continues to show your custom selection, and the correct selection shows up in App Settings > App Icon
* Check you can still manually switch to Spectrum 22 in App Settings, and to Cool Blue.

## Regression Notes

1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
